content/guides/10-minute-tutorial.md

```diff
@@ -792,12 +793,281 @@ Feature: Is it Friday yet?
 
 Congratulations! You've got your first green Cucumber scenario.
 
+# Evaluate a scenario
+Now that we've forced a scenario to pass, let's write one that passes or fails based on logic. Let's update our statement to evaluate whether or not `today` is equal to `"Friday"`.
+
+{{% block "javascript" %}}
+```javascript
+function isItFriday(today) {
+  return (today === "Friday") ? "TGIF" : "Nope";
+}
+```
+{{% /block %}}
+
+{{% block "ruby" %}}
+```ruby
+def is_it_friday?(day)
+  answer = (day == 'Friday') ? 'TGIF' : 'Nope'
+end
+```
+{{% /block %}}
+
+Run Cucumber again:
+
+{{% block "javascript" %}}
+```shell
+...
+1 scenario (1 passed)
+3 steps (3 passed)
+0m00.001s
+```
+{{% /block %}}
+
+{{% block "ruby" %}}
+```shell
+Feature: Is it Friday yet?
+  Everybody wants to know when it's Friday
+
+  Scenario: Sunday isn't Friday        # features/is_it_friday_yet.feature:4
+    Given today is Sunday              # features/step_definitions/stepdefs.rb:8
+    When I ask whether it's Friday yet # features/step_definitions/stepdefs.rb:12
+    Then I should be told "Nope"       # features/step_definitions/stepdefs.rb:16
+
+1 scenario (1 passed)
+3 steps (3 passed)
+0m0.049s
+```
+{{% /block %}}
+
+# Multiple scenarios
+We've seen what happens when it *isn't* Friday, but what about when it *is* Friday? Fortunately, we've already written the statement to evaluate whether or not `today` is equal to `"Friday"` and return `"TGIF"` if it is. So, let's write a scenario where today is Friday!
+
+Update the `is-it-friday-yet.feature` file.
+```gherkin
+Feature: Is it Friday yet?
+  Everybody wants to know when it's Friday
+
+  Scenario: Sunday isn't Friday
+    Given today is Sunday
+    When I ask whether it's Friday yet
+    Then I should be told "Nope"
+
+  Scenario: Friday is Friday
+    Given today is Friday
+    When I ask whether it's Friday yet
+    Then I should be told "TGIF"
+```
+
+{{% block "javascript" %}}
+Update the `stepdefs.js` file.
+```javascript
+const assert = require('assert');
+const { Given, When, Then } = require('cucumber');
+
+function isItFriday(today) {
+  return (today === "Friday") ? "TGIF" : "Nope";
+}
+
+// These two Given functions are very similar but, as written, require separate functions
+Given('today is Sunday', function () {
```

Comment:
Is it possible to pass the day as a String here? (same question for Ruby) Update: I see below that there is, because we change this when using Scenario Outline. I'd like to consider an intermediary refactoring step, where first we make the stepdef take a variable and then change from 2 scenarios to 1 scenario outline. My reasoning is that way we *explicitly* show a refactoring step in the code and usig scenario outline instead of scenario, but each separately (keeping in mind new users).